tests/tests_app/utilities/packaging/test_lightning_utils.py

```diff
@@ -10,17 +9,17 @@
     _verify_lightning_version,
 )
 
-
-def test_prepare_lightning_wheels_and_requirement(tmpdir):
```

Comment:
I think we should keep the test

Reply:
Any idea how to fix it? We can open a PR with uncommenting this test right after this is merged...

Reply:
Why did we leave the test commented out though? If we need to skip a test, we would add the skip mark instead. This is would be visible in the test output summary.